page_objects/menu_page.js

```diff
@@ -36,18 +36,18 @@ class MenuPage extends BasePage {
     getProductElement() {
         return  new Button(element(productLocator), 'Summer Dress Tab');
     };
-    getAddProduct1Element() {
-        return  new Button(element(addtocart1Locator), 'Summer Dress1');
-    };
-    getAddProduct2Element() {
-        return  new Button(element(addtocart2Locator), 'Summer Dress2');
-    };
-    getAddProduct3Element() {
-        return  new Button(element(addtocart3Locator), 'Summer Dress3');
+    getAddProductElement(index) {
+        let finalLocator =  Object.assign({}, addtocartLocator);
+        console.log('test'+ JSON.stringify(finalLocator));
+        finalLocator.value = finalLocator.value.replace("INDEX", index);
+        console.log('test'+ JSON.stringify(finalLocator));
+        console.log('test2'+ JSON.stringify(addtocartLocator));
```

Comment:
Please remove all console logs, they were just for debugging needs

Reply:
fixed